fastlane/metadata/en-CA/subtitle.txt

```diff
@@ -1 +0,0 @@
-Design a site, build a blog
```

Comment:
This and all the other `en-CA` files were deleted because `gp_downloadmetadata`

```
[22:02:19]: ---------------------------------
[22:02:19]: --- Step: gp_downloadmetadata ---
[22:02:19]: ---------------------------------
[22:02:19]: Project URL: https://translate.wordpress.org/projects/apps/ios/release-notes/
[22:02:19]: Locales: {"ar"=>"ar-SA", "da"=>"da", "de"=>"de-DE", "en-au"=>"en-AU", "en-ca"=>"en-CA", "en-gb"=>"en-GB", "es"=>"es-ES", "es-mx"=>"es-MX", "fr"=>"fr-FR", "he"=>"he", "id"=>"id", "it"=>"it", "ja"=>"ja", "ko"=>"ko", "nl"=>"nl-NL", "nb"=>"no", "pt-br"=>"pt-BR", "pt"=>"pt-PT", "ru"=>"ru", "sv"=>"sv", "th"=>"th", "tr"=>"tr", "zh-cn"=>"zh-Hans", "zh-tw"=>"zh-Hant"}
[22:02:19]: Source locale: -
[22:02:19]: Path: /Users/gio/Developer/a8c/wpios/fastlane/metadata
Downloading language: [...]
[22:02:27]: No translation available for en-CA
```

Notice, though, that those sources exist in GlotPress, it's only the release notes that are not there yet.

I _think_ the issue had to do with the HTTP response, likely the same 429s we've seen recently, but reported differently because in `gp_downloadmetadata` as opposed to our new automation. I haven't dug into it, though.

I'll keep an eye out for this tomorrow, when I finalize the release.